source/MapSalesPanel.h

```diff
@@ -62,8 +64,8 @@ class MapSalesPanel : public MapPanel {
 	void DrawInfo() const;
 
 	bool DrawHeader(Point &corner, const std::string &category);
-	void DrawSprite(const Point &corner, const Sprite *sprite) const;
-	void Draw(Point &corner, const Sprite *sprite, bool isForSale, bool isSelected,
+	void DrawSprite(const Point &corner, const Sprite *sprite, int swizzle = 0) const;
```

Comment:
Require the caller to specify the swizzle:
```suggestion
	void DrawSprite(const Point &corner, const Sprite *sprite, int swizzle) const;
```